src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router'
+import React from 'react'
+
+const Breadcrumbs = props => {
```

Comment:
I'm relatively ambivalent about it but I could see how that might help with understanding the component behavior. If you prefer that route, I'm game to give it a shot